src/structures_node.jl

```diff
@@ -107,7 +120,29 @@ struct RefStorage{T} <: Storage
     stor_res::T
     input::Dict{<:Resource, <:Real}
     output::Dict{<:Resource, <:Real}
-    data::Array{<:Data}
+    data::Vector{<:Data}
```

Comment:
Was there a reason why this isn't just `data::Vector{Data}` as for the other nodes?

Reply:
Yes. The parametric type is requiring it. It would not work otherwise.